core/src/autogluon/core/scheduler/mo_hyperband.py

```diff
@@ -21,28 +18,28 @@
 
 class MOHyperbandScheduler(HyperbandScheduler):
```

Comment:
What is the difference to MOASHAScheduler? Why do we need both?

Reply:
This looks like legacy code. If there is a good reason to have two different MO extensions of async Hyperband (or ASHA), please say that clearly in the comments. Otherwise, please bring them together.

In terms of naming, ASHA is pretty specific for "Hyperband promotion", so I usually prefer async Hyperband, or even async successive halving, since virtually nobody uses >1 bracket in the async case.